src/APIFunctions/Speaker.js

```diff
@@ -0,0 +1,83 @@
+import axios from 'axios';
+import { ApiResponse } from './ApiResponses';
+
+let PERIPHERAL_API_URL = process.env.REACT_APP_PERIPHERAL_API_URL
+  || 'http://localhost:8081/peripheralapi';
+
+
+console.debug(PERIPHERAL_API_URL);
```

Comment:
remove all console statements in this file, every api should also be sending a token